data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -94,9 +96,12 @@ public KustoOperationResult execute(String database, String command, ClientReque
         try {
             return new KustoOperationResult(response, clusterEndpoint.endsWith("v2/rest/query") ? "v2" : "v1");
         } catch (KustoServiceQueryError e) {
-            throw new DataServiceException(clusterEndpoint,
-                    "Error found while parsing json response as KustoOperationResult:" + e.getMessage(), e, e.isPermanent());
-        } catch (Exception e){
+            throw new DataServiceException(
+                    clusterEndpoint,
+                    "Error found while parsing json response as KustoOperationResult:" + e.getMessage(),
+                    e,
+                    e.isPermanent());
```

Comment:
Is this formatting approach really an improvement?
I think sometimes it is, but I'd argue often it isn't, in which case I don't think it should be enforced.

Reply:
The point of the formatting is not to be an improvement for every particular case, but to be *consistent*.
We decided on a limit of 160 characters per line, both this and the next sample are over it, that's why we split it.

Reply:
I agree that there are many things we want to make consistent. I'm not sure all things need to be consistent, though.

In this case, there are 2 options if a line is more than 160 chars:
1) Spill the remainder over to 1 new line (assuming the remainder is less than 160 chars, else a 3rd line, etc)
2) Change the entire signature such that each parameter gets its own line

I'm saying that I don't think (2) is an improvement over (1).